pyveg/paper/paper.md

```diff
@@ -129,6 +135,12 @@ Early warning signals are also computed using the `ewstools` package [@ewstools]
 including Lag-1 autocorrelation and standard deviation moving window plots.
 A sensitivity and significance analysis is also performed in order to determine
 whether any trends (quantified by Kendall tau values) are statistically significant.
+The vegetation return rate is calculated by fitting the crystall ball function
+to the offset50 and NDVI time series.
+
+Following data processing, `pyveg` is able to calculate summary plots using
+`pyveg_analysis_summary_data` which will accept summary statistics that are
```

Comment:
i would replace "will accept summary statistics that are..." to something like "which uses as input a collection of summary statistics extracted from the time series of each individual location obtained with the download and analysis functionality described above."